Makefile

```diff
@@ -415,3 +415,9 @@ ADMIN_HELPER_VERSION = 0.0.12
 		 -e 's/__HELPER_SCRIPT_CHECKSUM__/'$(HELPER_SCRIPT_HASH)'/g' \
 		 -e 's/__ADMIN_HELPER_VERSION__/'$(ADMIN_HELPER_VERSION)'/g' \
 	     $< >$@
+
+.PHONY: macos-precheck
+macos-precheck:
+ifeq ($(GOOS),darwin)
+	@command -v pkg-config >/dev/null 2>&1 || { echo >&2 "Error: pkg-config is not installed. Please run 'brew install pkg-config'."; exit 1; }
```

Comment:
Are you sure this is needed? I tested `go vet` after removing libvirt packages, and I made `pkg-config` non executable, and `go vet` succeeded.

Reply:
What errors do you get with `go vet -tags "containers_image_openpgp,build" ./...` if pkg-config is missing?

Reply:
```
bpalm at bpalm-mac in ~
$ brew uninstall pkg-config
Uninstalling /opt/homebrew/Cellar/pkgconf/2.5.0... (28 files, 520.8KB)

bpalm at bpalm-mac in ~/Repositories/go/src/github.com/crc-org/crc on macos_build_support [?]
$ go vet -tags "containers_image_openpgp,build" ./...

bpalm at bpalm-mac in ~/Repositories/go/src/github.com/crc-org/crc on macos_build_support [?]
$ go vet ./...
package github.com/crc-org/crc/v2/cmd/crc-embedder
	imports github.com/crc-org/crc/v2/cmd/crc-embedder/cmd
	imports github.com/crc-org/crc/v2/pkg/crc/machine/libvirt: build constraints exclude all Go files in /Users/bpalm/Repositories/go/src/github.com/crc-org/crc/pkg/crc/machine/libvirt
```

Should we add a make path that wraps `go vet -tags "containers_image_openpgp,build" ./...` ?

Reply:
> Should we add a make path that wraps go vet -tags "containers_image_openpgp,build" ./... ?

That would be `go vet --tags "build $(BUILDTAGS)"`, but since that’s already covered by `make lint`, so this `vet` Makefile target would not be used much imo. I’m not going to nack it though if someone has a strong need for it.